sqlalchemy_redshift/__init__.py

```
@@ -1,14 +1,17 @@
-from pkg_resources import DistributionNotFound, get_distribution, parse_version
+from importlib.metadata import version, PackageNotFoundError
+from packaging.version import parse as parse_version
 
-for package in ['psycopg2', 'psycopg2-binary', 'psycopg2cffi']:
+MIN_PSYCOPG2_VERSION = parse_version('2.5')
+
+for package in ['psycopg2', 'psycopg2-binary']:
     try:
-        if get_distribution(package).parsed_version < parse_version('2.5'):
+        if parse_version(version(package)) < parse_version(MIN_PSYCOPG2_VERSION):
             raise ImportError('Minimum required version for psycopg2 is 2.5')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)